<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>539</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>26</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cadastro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(75, 114, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Alunos</string>
     </attribute>
     <widget class="QLineEdit" name="inputcomplementoaluno">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>260</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	
background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Complemento</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputcsenhaaluno">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>310</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 8pt &quot;MS Shell Dlg 2&quot;;

background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Senha</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputsenhaaluno">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 8pt &quot;MS Shell Dlg 2&quot;;

background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Senha</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="labelturno">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>403</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>UD Digi Kyokasho NK-R</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TURMA:</string>
      </property>
     </widget>
     <widget class="QLabel" name="erro">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>135</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="turmabox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>404</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>101</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>102</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>103</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="inputsobrenomealuno">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Sobrenome</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputcpfaluno">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 8pt &quot;MS Shell Dlg 2&quot;;

background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="inputMask">
       <string>00000000000</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputnomealuno">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	
background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputenderecoaluno">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	
background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Endereço</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-100</x>
        <y>0</y>
        <width>531</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Sem Título-6.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="labelturno_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>404</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>UD Digi Kyokasho NK-R</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>CURSO:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cursobox">
      <property name="geometry">
       <rect>
        <x>391</x>
        <y>405</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inglês</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Espanhol</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="cadastraraluno">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>450</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 127);
border-radius:10px
</string>
      </property>
      <property name="text">
       <string>Cadastrar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputusuarioaluno">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	
background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Usuario</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputdatadenascimentoaluno">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>360</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	
background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="inputMask">
       <string>00/00/0000</string>
      </property>
      <property name="text">
       <string>//</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Professor</string>
     </attribute>
     <widget class="QLineEdit" name="inputsobrenomeprofessor">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Sobrenome</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputcpfprofessor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 8pt &quot;MS Shell Dlg 2&quot;;

background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="inputMask">
       <string>00000000000</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputsenhaprofessor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 8pt &quot;MS Shell Dlg 2&quot;;

background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Senha</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="cadastrarprofessor">
      <property name="geometry">
       <rect>
        <x>273</x>
        <y>450</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 127);
border-radius:10px
</string>
      </property>
      <property name="text">
       <string>Cadastrar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>-100</x>
        <y>0</y>
        <width>531</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Sem Título-6.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputcsenhaprofessor">
      <property name="geometry">
       <rect>
        <x>302</x>
        <y>310</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 8pt &quot;MS Shell Dlg 2&quot;;

background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Senha</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputnomeprofessor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">	
background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputcomplementoprofessor">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>260</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	
background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Complemento</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputenderecoprofessor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	
background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Endereço</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="labelturno2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>395</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>UD Digi Kyokasho NK-R</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>TURMAS DESIGNADAS:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="t101">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>406</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>101 MATUTINO</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="t102">
      <property name="geometry">
       <rect>
        <x>362</x>
        <y>406</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>102 VESPERTINO</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="t103">
      <property name="geometry">
       <rect>
        <x>487</x>
        <y>407</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>103 NOTURNO</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputusuarioprofessor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 8pt &quot;MS Shell Dlg 2&quot;;

background-color: rgb(255, 255, 255);
border-radius:10px</string>
      </property>
      <property name="text">
       <string>Usuario</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="aviso">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>370</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelturno2_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>UD Digi Kyokasho NK-R</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>MATERIA:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="materiaprofessor">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>450</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Portugues</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Matematica</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Geografia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Filosofia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Historia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Biologia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fisica</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
